cooker/lib/pages.js

```diff
@@ -255,6 +280,35 @@ function parseMarkdown(file, fileInfo) {
   return mdData;
 }
 
+/**
+ * Serves as a wrapper around various data resolution methods. Takes a fileInfo and returns an object that contains all of the variables contained within.
+ * 
+ * @param {Array} localeFiles - An array of locale files for consumption
+ */
+async function getDataFromLocaleFiles(localeFiles) {
+  let dataReturn = {}
+
+  // Currently there's no explicit order to this data, should we assume one? Question for later
+  for(const file of localeFiles) {
+    const localeInfo = path.parse(file);
```

Comment:
`parse` returns the extension, no need for the next line.
```
// Returns:
// { root: '/',
//   dir: '/home/user/dir',
//   base: 'file.txt',
//   ext: '.txt',
//   name: 'file' }
```

Reply:
Hah nice catch. this was a leftover from when I was using the filename less the extension here as well.